app/services/analytics_events.rb

```diff
@@ -226,8 +226,8 @@ def backup_code_created(enabled_mfa_methods_count:, **extra)
   end
 
   # Tracks when the user visits the Backup Code Regenerate page.
-  def backup_code_regenerate_visit(**properties)
-    track_event('Backup Code Regenerate Visited', **properties)
+  def backup_code_regenerate_visit(**analytics_properties_for_visit)
```

Comment:
Could you also incorporate the feedback from https://github.com/18F/identity-idp/pull/8940#discussion_r1290498798 re: listing out the expected properties?

Reply:
Yes. Thank you that's added in.